docs/virtualenv_guide.md

```diff
@@ -23,6 +23,18 @@ sudo apt-get install python3.6
 brew install python
 brew install python3
 ```
+* <b>Note:</b>
+Since we currently only support up to Python 3.6.X, you may need to brew install this older Python version. If you have had Python 3.6 installed previously, run the following:
```

Comment:
I would say that we support Python 3.6.X, not "up to", since we don't support python 3.5.X for example.